src/Nethermind/Ethereum.Transaction.Test/TransactionTests.cs

```diff
@@ -174,7 +174,7 @@ private void RunTest(TransactionTest test, IReleaseSpec spec)
 
             bool useChainId = transaction.Signature.V > 28UL;
 
-            TxValidator validator = new(useChainId ? ChainId.Mainnet : 0UL);
+            TxValidator validator = new(useChainId ? NetworkId.Mainnet : 0UL);
```

Comment:
Arent this suppose to be ChainId?

Reply:
NetworkId class is a list of predefined ids, all of these ids a both NetworkId and ChainId for their networks. Sometimes it's used as NetworkId, sometimes as ChainId. Still class name may seem misleading. Any ideas of better naming?
`NetworkAndChainIds`? `BlockchainIds`?

Reply:
See https://github.com/NethermindEth/nethermind/pull/4850/files#diff-31d7b5d9b31ad8f32e8c7a5912e00af3768c6bed71c5d80fd4b45b924339e201R30

Reply:
Let it be BlockchainIds
@MarekM25 @asdacap let me know if you have better ideas, will make it ``BlockchainIds` for now

Reply:
I mean the one passed into TxValidator, shouldn't that always be a chainid?

Reply:
https://eips.ethereum.org/EIPS/eip-155 tells signing should consider chainid